.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,10 +1,40 @@
 ---
-name: Feature request
-about: Suggest an idea for this project
+name: Feature Request
+about: Suggest an idea that should be added to the language.
 title: ''
 labels: feature request
 assignees: ''
 
 ---
 
+**Proposal:**
 
+**Example Code/Usage of your proposal:**
+
+**Example Code/Usage without your proposal (if applicable):**
+
+=====Delete message past this line before submitting the issue=====
+Wrap all code or terminal output between code blocks like so:
+
+```
+//I am text wrapped inside a code block.
+
+//Multiline is possible
+
+fn some_function() {
+
+}
+```
+
+or
+
+```
+root@v-machine$ make all
+rm -rf vc/
+git clone --depth 1 --quiet https://github.com/vlang/vc
+cc -std=gnu11 -w -o v vc/v.c -lm
+Self rebuild (22f162c => 735336e)
+rm -rf vc/
```

Comment:
Please replace these lines with `...` Having them in each template, may lead to false positives when searching issues later (because I am sure that many people would still not care to read the templates and will leave them as they are)